ocaml/xapi/message_forwarding.ml

```diff
@@ -1346,6 +1346,16 @@ functor
         else
           local_fn ~__context
 
+      let clear_vif_reservation ~__context ~vm =
+        Db.VM.get_VIFs ~__context ~self:vm
+        |> List.iter (fun vif ->
+               let vf = Db.VIF.get_reserved_pci ~__context ~self:vif in
+               Db.VIF.set_reserved_pci ~__context ~self:vif ~value:Ref.null ;
+               if Db.is_valid_ref __context vf then
+                 Db.PCI.set_scheduled_to_be_attached_to ~__context ~self:vf
+                   ~value:Ref.null
```

Comment:
This is redundant now, as also done in `clear_reservations`. But we can leave it, just in case...